### DISCUSSION TITLE
System Tray not showing on Ubuntu 20.04.1

### DISCUSSION THREAD

**Discussion by toabm** [hluk/CopyQ#1546] [General]
The icon is missing. CopyQ server running and working properly from command line but no trace of the tray icon...:(

**Comment by djmills64** [comment]
I'm getting the same.  When I first installed CopyQ it would show in the system tray after logging in.  But then a few weeks ago it stopped.  The process is running and it does collect copied text.  If I kill the process and then restart it will appear in the system tray.
Is there anything I can do to get it to appear by default (again)?

**Comment by hluk** [comment]
Can you use the workaround in https://github.com/hluk/CopyQ/issues/1526#issuecomment-756970566?

**Comment by toabm** [comment]
I actually fixed it by using other workaround I found in the net. The solution is similar thow, to add some delay to the AutoStart command. If you want me to try the exact solution you proposed I can do it too. I attach my solution on Ubuntu 20.04:

Kind regards,


Ángel Blanco Muñoz

Ingeniero Superior en Telecomunicaciones

Diseñador WEB

+34 686 022 822
 

    En martes, 26 de enero de 2021 06:48:13 CET, Lukas Holecek <notifications@github.com> escribió:  
 
 


Can you use the workaround in #1526 (comment)?

**Comment by djmills64** [comment]
I found 2 suggested solutions on the web:
* Add "sleep 30;" to the startup application command line - this didn't
work for me
* Add a delay line to the .desktop file - this worked


Cheers,

David

Mob: +61-4-11513404
